content/Arduino Cloud/Arduino SIM Card/How-to-connect-the-MKR-GSM-1400-to-Arduino-Cloud.md

```diff
@@ -27,13 +27,13 @@ The micro UFL port and the SIM tray are highlighted respectively in the image be
 
 <a id="set-up-instructions"></a>
 
-## Set up with IoT Cloud
+## Set up with Arduino Cloud
 
 ### 1. Add Arduino MKR GSM 1400 as a device
 
 1. Connect the board to your computer.
 
-2. Go to [Arduino IoT Cloud](https://app.arduino.cc/devices)
+2. Go to [Arduino Cloud](https://app.arduino.cc/devices)
```

Comment:
Since the next step is telling me to click the Devices tab, I'm expecting the Cloud Home page here.

```suggestion
2. Go to [Arduino Cloud](https://app.arduino.cc/)
```